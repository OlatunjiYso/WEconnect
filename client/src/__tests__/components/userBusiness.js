import React from "react";

import UserBusiness from '../../components/userBusiness';

describe('All tests for user business card component', () => {
 
  const props = {
   userBusiness: {
    image:'', name: '', id:0,
    }
  };
  
    it('should be defined', () => {
      expect(UserBusiness).toBeDefined();
    });
    it('should render the business card', () => {
      const { wrapper } = shallow(<UserBusiness {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
});
