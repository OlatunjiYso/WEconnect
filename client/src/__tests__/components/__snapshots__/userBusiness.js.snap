// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests for user business card component should render the business card 1`] = `undefined`;
